packages/suite/src/components/suite/modals/ImportTransaction/components/ExampleCSV.tsx

```diff
@@ -66,11 +68,15 @@ export const ExampleCSV = () => {
                 {isExpanded && (
                     <ExpandWrapper {...motionAnimation.expand}>
                         {/* CSV keys shouldn't be translated */}
-                        <P size="small">address,amount,currency</P>
+                        <P size="small">address,amount,currency{labelingAvailable && ',label'}</P>
                         <P size="small">
                             {addresses[0]},0.31337,{account.symbol.toUpperCase()}
+                            {labelingAvailable && ',Important transaction'}
+                        </P>
+                        <P size="small">
+                            {addresses[1]},0.1,USD
+                            {labelingAvailable && ',Sending coins to mother'}
```

Comment:
I am not sure with the labels. Maybe use some more neutral one?

Reply:
:D could be, thats for @Hannsek, also maybe it should be translated?

Reply:
@mroz22 Let's put there "Savings" and "Rent"

Reply:
38f619579353ba4ad8ed1386e4a4282059f3f738 also dropped the commit decreasing font size